lib/metadata/index.js

```diff
@@ -13,6 +13,26 @@ var filenames = require('./filenames'),
     funk = require('../funk'),
     github = require('../github');
 
+function inferDownstreamReview(metadata, content) {
+    var login = metadata.author.login;
+
+    if (login == "chromium-wpt-export-bot") {
+        return "Chromium";
+    } else if (login == "servo-wpt-sync") {
+        return "Servo";
+    } else if (login == "moz-wptsync-bot" ||
+        (login == "jgraham" && content.indexOf("MozReview-Commit-ID") > -1) ||
+        (login == "dbaron" && content.indexOf("Sync Mozilla CSS tests") > -1)) {
+        return "Firefox";
+    } else if (content.indexOf("WebKit export") > -1 &&
+        (login == "fwang" || login == "ms2ger" || login == "rniwa" ||
```

Comment:
Do we need to check `login` at all? Anyone can upstream from a WebKit patch. (I've done it, once.)

Reply:
Hmm, nevermind. I see now that this is not new. Though it'd be nice if WebKit had a better setup here. :) cc @youennf

Reply:
Perhaps in addition to the brittle title matching, WebKit exports should also use a dedicated label like `webkit-export` (similar to what Chromium & Gecko are currently doing). Only people with write access to the repo can set labels, so we have a lot more trust in labels than titles.

Reply:
Would you mind filing a WebKit bug about that?

Reply:
Ping @jugglinmike or @zcorpan

Reply:
And I've filed https://github.com/web-platform-tests/wpt-pr-bot/issues/100.